test/integration_test.go

```diff
@@ -2684,6 +2684,9 @@ func (ts *IntegrationTestSuite) TestDeterminismUpsertSearchAttributesConditional
 	options.SearchAttributes = map[string]interface{}{
 		"CustomKeywordField": "unset",
 	}
+	// TODO(cretz): There is a bug with search attribute names on standard
+	// visibility with eager workflow start
+	options.EnableEagerStart = false
```

Comment:
https://github.com/temporalio/temporal/issues/5119 (and https://github.com/temporalio/temporal/pull/5124)